skt/executable.py

```diff
@@ -610,6 +751,13 @@ def main():
     global retcode
 
     parser = setup_parser()
+
+    # If the user ran 'skt' with no arguments or subcommands, print the help
+    # text rather than printing "error: too few arguments".
+    if len(sys.argv[1:])  == 0:
```

Comment:
Besides having two spaces before the operator, pylint (which we don't run on the code yet) will complain about using `len()` in comparisons. Can we change this to `if not sys.argv[1:]:` instead?

Reply:
@veruu Sure, good catch! :)